Packaging/nix/devilutionx.metainfo.xml

```diff
@@ -64,7 +64,19 @@
 		</screenshot>
 	</screenshots>
 	<releases>
-		<release version="1.5.1" date="2023-xx-xx">
+		<release version="1.5.2" date="2024-01-01">
+			<description>
+				<p>This release includes the following updates:</p>
+				<ul>
+					<li>Multiplayer crash fixes</li>
+					<li>Improved packet validation for items</li>
+					<li>Game now builds on Ubuntu 22.04.<br>Warning: This may cause DevilutionX to not launch on systems with older Glibc versions!</li>

```

Comment:
I don't like this phrase at all
Any ideas how to word it better?

Reply:
"to fail to launch"?